Album popup is now album sidebar

danrabbit opened the pull request:
![screenshot from 2017-12-03 10 27 23](https://user-images.githubusercontent.com/7277719/33528411-c47ef744-d814-11e7-8c09-ffec6f1a9db7.png)

Bat41 commented:
A screenshot where the padding issue is visible:

![capture d ecran du 2017-12-03 19 32 54](https://user-images.githubusercontent.com/16254623/33528491-801dca6a-d861-11e7-84a9-a8e0ed5f3445.png)

danrabbit commented:
Fixed the wrapping issue and renamed file since it's no longer a dialog.

In my experience, Gtk.Revealer doesn't play nice with Gtk.Paned. Can we save that for another branch?
